drivers/network/afd/afd/connect.c

```diff
@@ -463,8 +463,66 @@ AfdStreamSocketConnect(PDEVICE_OBJECT DeviceObject, PIRP Irp,
         if( !FCB->RemoteAddress )
             Status = STATUS_NO_MEMORY;
         else
+        {
+	        /* original datagram code set STATUS_SUCCESS and exited here
+            // THIS CODE WAS COPIED FROM STREAM CODE BELOW, IT SHOULD BE SAFE AND 'CORRECT', BUT MAY NOT BE OPTIMAL. */
+            AFD_DbgPrint(MID_TRACE,("AfdStreamSocketConnect: calling WarmSocketConnection(FCB)\n"));
+	        if (FCB->State ==  SOCKET_STATE_CREATED) 
```

Comment:
There are still many EOL whitespaces in this PR. like in this line. Please use the Notepad++ feature to fix them all at once:
![image](https://github.com/reactos/reactos/assets/33393466/ce005897-cfe9-4bcd-ab34-dfa388b3a828)

Reply:
I made this change to all of the PR files, thanks for the tool suggestion.